### ISSUE TITLE
Understanding 2.4.11 - Confusing wording due to page background change

### ISSUE THREAD

**Issue by matatk** [w3c/wcag#2893]
[Figure 6 in the Understanding document for 2.4.11](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html#focus-indicator-star-with-light-focus), along with the text in the paragraph above, is confusing because it sounds like when the document was written, the page background was white. The text refers to the white background, and the "light gray" of the focus indicator (which I believe was intended to be just the square border that's shown in the figure).

However, the page background itself is now a very light grey, which makes this wording confusing, as the focus indicator now looks more like a solid white box, with a _dark_ grey border.

I think the cause of this is the new layout and styling for the Understanding page (which looks great, and clear IMHO :-)). If that's the case, then there may well be other similar issues.

Note: The text above the figure actually refers to "mid gray" which I think should be adopted in the caption, as that would help with consistency generally.
